api/room/schema.py

```diff
@@ -20,6 +21,16 @@ class Meta:
         model = RoomModel
 
 
+def save_room_tags(room, room_tags):
+    for tag in room_tags:
+        room_tag = TagModel.query.filter_by(id=tag).first()
+        if not room_tag:
+            room.delete()
+            raise GraphQLError("Tag not found")
+        room.room_tags.append(room_tag)
+        room.save()
+
```

Comment:
Good work @bevkololi. But I am of the opinion that there can be a better way to refactor this to avoid calling save multiple times?

Reply:
Yes, thanks @kampkelly, as I had already saved it in the save_room_tags() method i do not need to save it again. Updating this as we speak.